redditnav.js

```diff
@@ -125,21 +125,32 @@ $(function() {
     });
   });
 
-
+  var released = true;
 
   $(document).keydown(function(e) {
-	if (!$(e.target).is('input, textarea')) {
-		var pos = $(window).scrollTop();
-		console.log(pos);
-
-		if(e.keyCode == 81){
-			e.preventDefault();
-			goToNextParent(pos, UP);
-		}
-		else if (e.keyCode == 87){
-			e.preventDefault();
-			goToNextParent(pos, DOWN);
-		}
-	}
+    if (e.repeat != undefined) {
+      released = false;
+    }
+    if (!released) {
+      return;
+    }
+    released = false;
+  	if (!$(e.target).is('input, textarea')) {
```

Comment:
Here's a fix that would work:
https://gist.github.com/justinthec/54082fe60dd2ce8da2e6